freshdesk/v1/test.py

```diff
@@ -254,11 +254,11 @@ def test_get_contact(self):
     def test_list_contacts(self):
         contacts = self.api.contacts.list_contacts()
         self.assertIsInstance(contacts, list)
```

Comment:
Maybe a bit pedantic, but do you want to change these `assertEqual`'s to plain `assert`'s for consistency?

Reply:
Sure, why not.